neo/Cryptography/ECC/ECDsa.cs

```diff
@@ -36,9 +43,8 @@ private BigInteger CalculateE(BigInteger n, byte[] message)
 
         public BigInteger[] GenerateSignature(byte[] message)
```

Comment:
This function is RISKY.
Private Key leaks because of this function.

Recommend to remove this function or rewrite it.